supervision/keypoint/core.py

```diff
@@ -101,7 +102,46 @@ def __eq__(self, other: KeyPoints) -> bool:
         )
 
     @classmethod
-    def from_inference(cls, inference_result: dict) -> KeyPoints:
+    def from_inference(cls, inference_result: Union[dict, Any]) -> KeyPoints:
+        """
+        Create a `sv.KeyPoints` object from the [Roboflow](https://roboflow.com/)
+        API inference result or the [Inference](https://inference.roboflow.com/)
+        package results. When a keypoint detection model is used, this method
+        extracts the keypoint coordinates, class IDs, confidences, and class names.
+
+        Args:
+            inference_result (dict, any): The result from the
+                Roboflow API or Inference package containing predictions with keypoints.
+
+        Returns:
+            (KeyPoints): A KeyPoints object containing the keypoint coordinates,
+                class IDs, and confidences of each keypoint.
+
+        Example:
+            ```python
+            import cv2
+            import supervision as sv
+            from inference import get_model
+
+            image = cv2.imread(<SOURCE_IMAGE_PATH>)
+            model = get_model(model_id="yolov8s-640")
```

Comment:
The problem with this doc string is that `"yolov8s-640"` is an object detection model.

Reply:
Good catch. Putting a placeholder `"<POSE_MODEL_ID>"` until we have a well-trained COCO human-pose on prod.